src/pages/Homepage/Homepage.jsx

```diff
@@ -0,0 +1,71 @@
+import React , { useState , useEffect } from 'react'
+import "./Homepage.css"
+import axios from "axios"
+import OnePlus from "../../images/OnePlus.jpg"
+import {Nav} from "../../components/Nav/Nav"
+import { Link } from "react-router-dom"
+
+function Homepage() {
+
+const [categories,setCategories] = useState([]);
+
+useEffect(() => {
+(async function fetchCategories() {
+try{
+const resp = await axios.get("/api/categories")
+console.log(resp);
+setCategories(resp.data.categories)
+}
+catch(err) {
+console.log(err);
```

Comment:
- remove console.log from the file 
- In case of error you can show something on screen, user won't check console so he need to know when some error occurred from catch

Reply:
done